common/functions/src/scalars/arithmetics/arithmetic_mul.rs

```diff
@@ -33,8 +34,154 @@ impl ArithmeticMulFunction {
             .features(FunctionFeatures::default().deterministic().monotonicity())
     }
 
-    pub fn get_monotonicity(_args: &[Monotonicity]) -> Result<Monotonicity> {
-        //TODO
-        Ok(Monotonicity::default())
+    pub fn get_monotonicity(args: &[Monotonicity]) -> Result<Monotonicity> {
+        arithmetic_mul_div_monotonicity(args, DataValueArithmeticOperator::Mul)
+    }
+}
+
+pub fn arithmetic_mul_div_monotonicity(
+    args: &[Monotonicity],
+    op: DataValueArithmeticOperator,
+) -> Result<Monotonicity> {
+    if args.len() != 2 {
+        return Err(ErrorCode::BadArguments(format!(
+            "Invalid argument lengths {} for get_monotonicity",
+            args.len()
+        )));
+    }
+
+    let err_opt = if let DataValueArithmeticOperator::Mul | DataValueArithmeticOperator::Div = op {
+        None
+    } else {
+        Some(Err(ErrorCode::BadArguments(format!(
+            "Invalid operator '{}' for get_monotonicity",
+            op
+        ))))
+    };
+    if let Some(err) = err_opt {
+        return err;
+    }
+
+    let flip_positive_by_operator = |positive: bool| -> bool {
+        match op {
+            DataValueArithmeticOperator::Mul => positive,
+            DataValueArithmeticOperator::Div => !positive,
+            _ => unreachable!(),
+        }
+    };
+
+    let f_x = &args[0];
+    let g_x = &args[1];
+
+    match (f_x.is_constant, g_x.is_constant) {
+        // both f(x) and g(x) are constant
+        (true, true) => Ok(Monotonicity::create_constant()),
+
+        //f(x) is constant
+        (true, false) => {
+            if f_x.gt_eq_zero()? {
```

Comment:
12/g(x), also need check the domain of g(x), when g(x)>0 decreasing, when g(x)<0 increasing.